src/Tizen.NUI/src/internal/Common/ProcessorController.cs

```diff
@@ -0,0 +1,97 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using Tizen.NUI.BaseComponents;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System;
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Singleton class.
+    /// ProcessorController handles ProcessorCallbacks those are requested to be called for each end of event process.
+    /// If there are works those required to be called after all of the event process for the efficiency, add them ProcessorController.
+    /// The calling order is not guaranteed but ProcessorCallbackes of LayoutController those are added by using AddLayoutControllerProcessor,
+    /// are called after ordinaly ProcessCallbacks added by AddProcessor.
+    ///
+    /// The added callbacks will be called every time of event tick.
+    /// If the callback is not be wanted to be called anymore, remove it.
+    /// </summary>
+    internal sealed class ProcessorController : Disposable
+    {
+        private ProcessorController() : this(Interop.ProcessorController.New(), true)
+        {
+        }
+
+        internal ProcessorController(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        private static ProcessorController instance = null;
+
+        public static ProcessorController Instance
+        {
+            get
+            {
+                if(instance == null)
+                {
+                    instance = new ProcessorController();
+                }
+                return instance;
+            }
+        }
+
+        /// <summary>
+        /// Add Processor callback on this ProcessorController.
+        /// The input processorCallback will be called end of this event tick.
+        /// </summary>
+        public void AddProcessor(ProcessorInterface.ProcessorCallback processorCallback)
+        {
+            Interop.ProcessorController.AddCallback(SwigCPtr, processorCallback);
+        }
+
+        /// <summary>
+        /// Add Processor callback of LayoutController on this ProcessorController.
+        /// The layoutControllerProcessorCallbacks will be called after other callbacks are finished.
+        /// </summary>
+        public void AddLayoutControllerProcessor(ProcessorInterface.ProcessorCallback layoutControllerProcessorCallback)
+        {
+            Interop.ProcessorController.AddLayoutControllerCallback(SwigCPtr, layoutControllerProcessorCallback);
+        }
+
+        /// <summary>
+        /// Remove Processor callback on this ProcessorController.
+        /// The removed callback will not be called anymore.
+        /// </summary>
+        public void RemoveProcessor(ProcessorInterface.ProcessorCallback processorCallback)
```

Comment:
I have a question. 
1) if user do `AddProcesser(userCallback)` and do NOT `RemoveProcessor(userCallback)`, then `userCallback` is called continuously?
2) many `processorCallback` can be added? like this? `ProcessorController.Instance.AddLayoutControllerProcessor(callback1)`, ... ,  `ProcessorController.Instance.AddLayoutControllerProcessor(callback_n)` ?

Reply:
1. Yes.
I wrote it on the comment of class above.
/// The added callbacks will be called every time of event tick.
/// If the callback is not be wanted to be called anymore, remove it.

2. Yes too. it is possible.